llvm/docs/RISCVUsage.rst

```diff
@@ -426,6 +426,12 @@ The current vendor extensions supported are:
 ``Xwchc``
   LLVM implements `the custom compressed opcodes present in some QingKe cores` by WCH / Nanjing Qinheng Microelectronics. The vendor refers to these opcodes by the name "XW".
 
+``xmipscmove``
```

Comment:
Capital X

Reply:
Thanks, addressed in https://github.com/llvm/llvm-project/pull/121394